paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PrimaryButton.kt

```diff
@@ -219,11 +234,11 @@ internal class PrimaryButton @JvmOverloads constructor(
     /**
      * Used to override the current UI state of the Primary Button
      */
-    internal data class UIState(
-        val label: String?,
-        val onClick: (() -> Unit)?,
+    internal data class UIState constructor(
+        val label: String,
+        val onClick: () -> Unit,
         val enabled: Boolean,
-        val visible: Boolean
+        val lockVisible: Boolean,
```

Comment:
Nice rename!

Reply:
It looks like that, but it’s two different things: `visible` is now modelled with nullability, and `lockVisible` was previously set explicitly when setting up the button.